src/_widths.css

```diff
@@ -51,23 +51,15 @@
 .w-10 {  width:  10%; }
 .w-20 {  width:  20%; }
 .w-25 {  width:  25%; }
-.w-30 {  width:  30%; }
-.w-33 {  width:  33%; }
```

Comment:
Not a fan of removing thirds. At the Spokesman-Review, a ton of pages use three up layouts. The 30, 40, etc., are less important, but being able to have 33% is useful.

Reply:
agreed. we use it a bit too.